src/Layouts/AppWrapper.tsx

```diff
@@ -5,6 +5,7 @@ import { ReactNode } from "react";
 import AppHeader from "./AppHeader";
 import { ReactComponent as GlobalSvg } from "../media/Icons/global.svg";
 import { ROUTE_LINKS } from "../Components/Routes";
+import backgroundImage from "../media/background-vector-image.svg";
```

Comment:
`background-vector-image` let's rename to just `background`.

Reply:
Renamed.